Example mod/Example mod.csproj

```diff
@@ -41,7 +41,7 @@
     <Reference Include="System.Data" />
     <Reference Include="System.Xml" />
     <Reference Include="UnityEngine.CoreModule">
-      <HintPath>..\Dependencies\UnityEngine.CoreModule.dll</HintPath>
+      <HintPath>..\..\DependenciesBZ\UnityEngine.CoreModule.dll</HintPath>
```

Comment:
**TODO** Separate Unity DLLs from assembly-csharp DLLs.  
The Unity libraries are now the same between both games and can be handled together.

Reply:
For now, Unity DLLs are simply in the Dependencies folder